new-docs/spelling/allowed_words.txt

```diff
@@ -113,6 +114,7 @@ pseudocode
 PyPI
 quickstart
 RBAC
+RC
```

Comment:
I'm not seeing this word anywhere in the proposed changes; do we need it?

Reply:
I think I misspelled GC and it immediately moved it down there lol.